editoast/src/models/stdcm_log.rs

```diff
@@ -8,10 +8,23 @@ use utoipa::ToSchema;
 
 use crate::core::stdcm::Request;
 use crate::core::stdcm::Response;
+use crate::error::InternalError;
 use crate::models::prelude::*;
 
 editoast_common::schemas! {
     StdcmLog,
+    StdcmResponseOrError,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+#[serde(rename_all = "snake_case")]
+// We accepted the difference of memory size taken by variants
+// Since there is only on success and others are error cases
+#[allow(clippy::large_enum_variant)]
+pub enum StdcmResponseOrError {
+    #[schema(value_type = StdcmResponse)]
+    Response(Response),
+    RequestError(InternalError),
```

Comment:
```suggestion
    RequestError(serde_json::Value),
```

for the split in crates plz